dispatch/static/manager/package.json

```diff
@@ -7,45 +7,45 @@
   "contributors": [
     "Ben Cook <b@bencook.ca>"
   ],
-  "devDependencies": {
-    "@blueprintjs/core": "^1.1.0",
-    "babel": "^6.5.2",
+  "dependencies": {
+    "@blueprintjs/core": "^1.9.0",
+    "babel": "^6.23.0",
     "babel-core": "^6.21.0",
     "babel-loader": "^6.2.10",
     "babel-preset-es2015": "^6.13.2",
     "babel-preset-react": "^6.11.1",
+    "css-loader": "^0.26.1",
     "draft-js": "^0.9.1",
     "es6-promise": "^3.2.1",
-    "gulp": "^3.9.0",
-    "gulp-clean": "^0.3.2",
-    "gulp-sass": "^2.3.1",
-    "gulp-sourcemaps": "^1.6.0",
-    "gulp-util": "^3.0.7",
+    "file-loader": "^0.10.0",
     "history": "^2.1.2",
     "immutable": "^3.8.1",
     "isomorphic-fetch": "^2.2.1",
     "jquery": "^3.1.1",
     "js-cookie": "^2.1.2",
+    "node-sass": "^4.5.0",
     "normalizr": "^2.2.1",
     "qwery": "^4.0.0",
     "ramda": "^0.22.1",
     "react": "^15.3.1",
     "react-addons-css-transition-group": "^15.4.1",
     "react-document-title": "^2.0.2",
     "react-dom": "^15.3.1",
-    "react-icons": "^2.2.1",
     "react-redux": "^4.4.5",
-    "react-redux-loading-bar": "^2.3.1",
+    "react-redux-loading-bar": "2.6.6",
     "react-router": "^2.7.0",
     "react-router-redux": "^4.0.5",
     "redux": "^3.5.2",
     "redux-promise-middleware": "^4.0.0",
     "redux-thunk": "^2.1.0",
+    "sass-loader": "^6.0.2",
```

Comment:
This distinction is also for when we install packages in a production setting, since packages under devDependency won't be install when we use the --production flag.